src/esm_runscripts/helpers.py

```diff
@@ -136,6 +137,78 @@ def assemble_log_message(
     return line
 
 
+def update_reusable_files(config, reusable_filetypes=None):
+    """
+    Reusable files, in general (``config["general"]["reusable_filetypes"]``), or model
+    specific (``config[<model>]["reusable_filetypes"]``) forces files of a given type,
+    such as ``bin``, ``input``, ``forcing``, etc., to be copied only once into the
+    general experiment folder, and then **reused** them through out every run. If a
+    given file type is not present in this list, that file type will be copied from
+    its external source (i.e. for binaries from the folder where the model is compiled).
+
+    This method takes care of removing the reusable file types from
+    ``reusable_filetypes``, specified by the user in the ``esm_runscripts`` call with
+    the ``--update-files`` flag.
+
+    Parameters
+    ----------
+    config : dict
+        Dictionary containing the information about the experiment.
+    reusable_filetypes : list
+        List of reusable file types that can be specified by the user.
+
+    Returns
+    -------
+    config : dict
+        If the user gives no ``reusable_filetypes`` as input. ``config`` contains the
+        changes requested by the user with the ``--update-flag``.
+    reusable_filetypes : list
+        If the user gives ``reusable_filetypes`` as input. ``reusable_filetypes``
+        contains the changes requested by the user with the ``--update-flag``.
+    """
+    # Set the logic for what to return
+    if reusable_filetypes:
+        return_config = False
+    else:
+        return_config = True
+        reusable_filetypes = config["general"].get("reusable_filetypes", [])
+
+    update_files = config["general"].get("command_line_config", {}).get("update_files")
+    potentially_reusable_files = config["general"]["potentially_reusable_files"]
```

Comment:
I'm not sure I understand this: is there a default for `config["general"]["potentially _reusable_files"]`?

Reply:
Yes, it is defined here:
https://github.com/esm-tools/esm_tools/blob/590a2181ea95c867c79f5a956ad8b56169fb1d83/src/esm_runscripts/prepare.py#L497-L500

Reply:
```suggestion
    # NOTE(MAM, PG): Originally defined in prepare.py
    # https://tinyurl.com/2p8awzsu
    potentially_reusable_files = config["general"]["potentially_reusable_files"]
```